roles/lnls-ans-role-cs-studio/defaults/main.yml

```diff
@@ -1,16 +1,24 @@
 ---
-csstudio_version: 4.6.1.25
-csstudio_repository: production
-csstudio_base_url: "https://artifactory.esss.lu.se/artifactory/CS-Studio"
-csstudio_archive: "{{ csstudio_base_url }}/{{ csstudio_repository }}/{{ csstudio_version }}/cs-studio-ess-{{ csstudio_version }}-linux.gtk.x86_64.tar.gz"
+
+csstudio_version: "{{ pkg_version_csstudio }}"
```

Comment:
I think we should use sensible defaults without depending on external roles variables. I know we already depend on external variables for `vars` directory, but we should strive for making generic roles, usable without the whole `lnls` environment.

Ansible will overwrite variables in `defaults` if they are set in `group_vars`,  `host_vars`, etc